Rock.Messaging.RabbitMQ/IRabbitSessionConfiguration.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Rock.Messaging.RabbitMQ
+{
+    public interface IRabbitSessionConfiguration
```

Comment:
If setters aren't necessary, they ought to be removed from our other implementations. 

Rabbit calls it vHost. I tried to remain consistent, did I miss a spot?